lib/Dialect/Openfhe/IR/OpenfheOps.td

```diff
@@ -58,6 +58,21 @@ class Openfhe_BinaryOp<string mnemonic, list<Trait> traits = []>
   let results = (outs NewLWECiphertext:$output);
 }
 
+class Openfhe_BinaryInPlaceOp<string mnemonic, list<Trait> traits = []>
+  : Openfhe_Op<mnemonic, traits # [
+    AllTypesMatch<["lhs", "rhs"]>,
+    ]> {
+
+  let summary = "In-place binary operation for OpenFHE";
+
+  let arguments = (ins
```

Comment:
```suggestion

  let arguments = (ins
```

The summary is irrelevant in the base class because it is overridden by all instantiated defs. I can be removed with no impact.